charts/cinder-csi-plugin/values.yaml

```diff
@@ -160,11 +160,12 @@ secret:
 #      ca-file=/etc/cacert/ca-bundle.crt
 
 storageClass:
-  enabled: true
```

Comment:
Okay, this needs a release note for sure to show that the options are moved. I think I'd also keep the global option here and treat individual options as ones overriding the global one.

Reply:
I have updated the PR with the release notes and kept the default with an override.